### ISSUE TITLE
[Localization] Lost language keys

### ISSUE THREAD

**Issue by TUsama** [Mithion/Mana-And-Artifice#200]
Here are the keys which don't show in the en_us.json
```	
	"item.minecraft.tipped_arrow.effect.minor_mana_potion":"",
	"item.minecraft.tipped_arrow.effect.light_mana_potion":"",
	"item.minecraft.tipped_arrow.effect.mana_potion":"",
	"item.minecraft.tipped_arrow.effect.greater_mana_potion":"",
	"item.minecraft.tipped_arrow.effect.superior_mana_potion":"",
	The potion arrow.

	"block.mana-and-artifice.transitory_tunnel": "",
	"block.mana-and-artifice.chalk_rune": "",  
The rune on the ground draw by wizard chalk.
	"block.mana-and-artifice.transitory_tile": "",

	

	"entity.mana-and-artifice.thrown_manaweave_potion": "", 
This is the entity of Manaweave in a Bottle being thrown.
	"entity.mana-and-artifice.sentry_projectile",
	"entity.mana-and-artifice.spell_entity_wall": "",
	"entity.mana-and-artifice.spell_entity_wave": "",
	"entity.mana-and-artifice.spell_entity_emanation": "",
	"entity.mana-and-artifice.spell_entity_projectile": "",
	"entity.mana-and-artifice.spell_fx": "",
	"entity.mana-and-artifice.spell_rune_entity": "",

	
```

**Comment by Mithion** [comment]
Hey sorry it took me a while to respond to this!

Thanks for the list!  I'll make sure I add them all for next patch.

**Comment by TUsama** [comment]
Ok, thanks a lot!😀

**Comment by Mithion** [comment]
This has been fixed in 1.4.0.0 and will go out with the next release.
